### ISSUE TITLE
cli: Support editing app metadata

### ISSUE THREAD

**Issue by lmars** [flynn/flynn#1483]
I should be able to manage app metadata via the CLI, for example:

```
$ flynn meta
KEY  VAL
foo  bar

$ flynn meta set foo=baz bar=qux

$ flynn meta
KEY  VAL
foo  baz
bar  qux

$ flynn meta unset foo

$ flynn meta
KEY  VAL
bar  qux
```

This will help with arbitrary tagging of resources (#1313).

**Comment by temujin9** [comment]
I'm picking this ticket up, since we need the result and it seems easy enough to accomplish. Ping here if you need to know status.

**Comment by titanous** [comment]
Implemented in #1618.
